src/Effects/Effects.iOS/Effects/Label/LabelCustomFont.cs

```diff
@@ -10,16 +10,29 @@ namespace FormsCommunityToolkit.Effects.iOS
 {
     public class LabelCustomFont : PlatformEffect
     {
+        private RoutingEffects.LabelCustomFont _effect;
         protected override void OnAttached ()
         {
             var control = Control as UILabel;
 
             if (control == null)
                 return;
 
-            var effect = (RoutingEffects.LabelCustomFont)Element.Effects.FirstOrDefault(item => item is RoutingEffects.LabelCustomFont);
-            if (effect != null && !string.IsNullOrWhiteSpace(effect.FontPath))
-                control.Font = UIFont.FromName(effect.FontFamilyName, control.Font.PointSize);
+            _effect = (RoutingEffects.LabelCustomFont)Element.Effects.FirstOrDefault (item => item is RoutingEffects.LabelCustomFont);
+            if (_effect != null && !string.IsNullOrWhiteSpace (_effect.FontPath))
+                control.Font = UIFont.FromName (_effect.FontFamilyName, control.Font.PointSize);
+
+            // After one of these properties change, reapply the custom font
+            // As per https://bugzilla.xamarin.com/show_bug.cgi?id=33666
+            Element.PropertyChanged += (sender, e) => {
+                if (e.PropertyName == Label.TextColorProperty.PropertyName) {
```

Comment:
Can we combine all checks? After all the executed statement is the same everywhere.
If not: Please put else on a new line instead of behind the brace.